File: csharp/MaxPainUI/ClientApp/src/app/models/message.ts
import { Pipe, PipeTransform } from '@angular/core';
import { DatePipe } from '@angular/common';
import { UtilsService } from '../services/utils.service';

export class Message {
  Id: Number;
  Subject: String;
  Body: String;
  CreatedOn: Date;

  constructor(private utils: UtilsService) { }

  get est(): Date {
    return this.utils.utcToEst(this.CreatedOn);
  }
}

File: csharp/MaxPainInfrastructure/Models/Message.cs
namespace MaxPainInfrastructure.Models
{
    public class Message
    {
        /*
        public string subject { get; set; }
        public string body { get; set; }
        public DateTime? createdOn { get; set; }
        */

        public long Id { get; set; }
        public string Subject { get; set; }
        public string Body { get; set; }
        public DateTime? CreatedOn { get; set; }
    }
}
